src/Nordea/Components/DropdownFilter.elm

```diff
@@ -217,10 +225,16 @@ view attrs (DropdownFilter config) =
                         [ width (pct 100)
                         , borderBottomLeftRadius (pct 0) |> Css.important |> styleIf config.hasFocus
                         , borderBottomRightRadius (pct 0) |> Css.important |> styleIf config.hasFocus
-                        , descendants [ typeSelector "input" [ paddingRight (rem 3) ] ]
+                        , descendants
+                            [ typeSelector "input"
+                                [ paddingRight (rem 3)
+                                , position relative |> Css.important |> styleIf config.hasSearchIcon
+                                , paddingLeft (rem 2) |> Css.important |> styleIf config.hasSearchIcon
```

Comment:
Instead of adding these styles here in L231 and L232, we can directly use `withSearchIcon` from `TextInput` above.
`|> TextInput.withSearchIcon config.hasSearchIcon`